opl/investigator/config.py

```diff
@@ -75,6 +75,9 @@ def load_config(conf, fp):
         assert not conf.history_es_server.endswith("/")
         conf.history_es_index = data["history"]["es_index"]
         conf.history_es_query = data["history"]["es_query"]
+        conf.history_es_server_user = data["history"]["es_server_user"]
```

Comment:
Hello! conf.history_es_server_user and pass needs to stay in that if because they will not be configured every time. Just conf.history_es_server_verify should be excluded from it because it can be used for unauthenticated ES.

Reply:
Maybe something like this:
```
    if conf.history_type == "elasticsearch":
        conf.history_es_server = data["history"]["es_server"]
        assert not conf.history_es_server.endswith("/")
        conf.history_es_index = data["history"]["es_index"]
        conf.history_es_query = data["history"]["es_query"]
        if "es_server_user" in data["history"]:
            conf.history_es_server_user = data["history"]["es_server_user"]
            conf.history_es_server_pass_env_var = data["history"]["es_server_pass_env_var"]
        if "es_server_verify" in data["history"]:
            conf.history_es_server_verify = data["history"]["es_server_verify"]
        else:
            conf.history_es_server_verify = True
```